### ISSUE TITLE
WGLMakie exportable and interactable Axis3

### ISSUE THREAD

**Issue by hammy4815** [MakieOrg/Makie.jl#3664]
Hi,

I noticed when filling an Axis3 using WGLMakie with data, I can serve the html and interact with it while the Julia server is running. However, when I export the html statically (using `Page(exportable=true, offline=true)` or just filling a `DOM.div()` object with my figure), and serve it, the Axis3 object becomes static. 

I can partially solve the problem by wrapping a custom camera around the object:

```julia
...
fig = Figure()
ax = fig[1,1] = Axis3(...)
Camera3D(ax.scene)
```

However, this only allows me to interact with the data being plotted (zoom in and rotate) while keeping the axis itself (ticks, labels, etc.) fixed. 

I'm also unsure if I should instead file this issue to Bonito.jl. 

Thanks

**Comment by SimonDanisch** [comment]
Yeah, this is a big TODO on the WGLMakie roadmap: https://github.com/MakieOrg/Makie.jl/issues/2977.
It will take quite some time to rewrite the Axis3 in JS.
